src/dec/entis/eri_image_decoder.cc

```diff
@@ -41,8 +42,8 @@ static image::EriHeader read_header(
 
     const s32 width         = input_stream.read_le<u32>();
     const s32 height        = input_stream.read_le<u32>();
-    header.width            = std::abs(width);
-    header.height           = std::abs(height);
+    header.width            = std::fabs(width);
+    header.height           = std::fabs(height);
```

Comment:
`std::abs` results in an ambigious overload apparently, so the workaround is to use `std::fabs` or std::abs with a cast to double beforehand.